<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonVisitsSetupDialog</class>
 <widget class="QDialog" name="PersonVisitsSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>&amp;Врач</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbPerson">
     <item>
      <property name="text">
       <string>Врач</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblEventPurpose">
     <property name="text">
      <string>&amp;Назначение обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbEventPurpose"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>&amp;Тип обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblScene">
     <property name="text">
      <string>&amp;Место</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbScene"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblWorkOrganisation">
     <property name="text">
      <string>Занятость</string>
     </property>
     <property name="buddy">
      <cstring>cmbWorkOrganisation</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="COrgComboBox" name="cmbWorkOrganisation"/>
   </item>
   <item row="8" column="4">
    <widget class="QToolButton" name="btnSelectWorkOrganisation">
     <property name="text">
      <string>...</string>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>По&amp;л</string>
     </property>
     <property name="buddy">
      <cstring>cmbSex</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="cmbSex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>Во&amp;зраст с</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="4">
    <widget class="QFrame" name="frmAge">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="edtAgeFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeTo">
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeYears">
        <property name="text">
         <string>лет</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>111</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>cmbEventPurpose</tabstop>
  <tabstop>cmbEventType</tabstop>
  <tabstop>cmbScene</tabstop>
  <tabstop>cmbWorkOrganisation</tabstop>
  <tabstop>btnSelectWorkOrganisation</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PersonVisitsSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PersonVisitsSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
